linter/orb.yaml

```diff
@@ -28,6 +28,11 @@ commands:
         description: >
           Pre-commit package version.
         type: string
+      pre_commit_args:
```

Comment:
I'm thinking we should just call this `args`; they're already definitely `pre-commit` related as the command is `pre-commit`. Applies to the usage in the job below, as well.